ksql-engine/src/test/java/io/confluent/ksql/integration/JsonFormatTest.java

```diff
@@ -187,6 +203,44 @@ public void testSinkProperties() throws Exception {
     assertThat(
         topicClient.describeTopics(ImmutableList.of(streamName)).get(streamName).partitions(),
         hasSize(3));
+    Map<ConfigResource, Config> configResourceConfigMap = topicClient.describeConfigs
```

Comment:
This block is used 3 times. Should be moved to a utility method named `verifyCleanupPolicy(String policy)`. Then you can get the property from the cluster and verify that it matches the expectation.